src/drive/web/modules/viewer/styles.styl

```diff
@@ -0,0 +1,57 @@
+@require 'settings/breakpoints.styl'
+@require 'settings/palette.styl'
+
+.pho-viewer-noviewer-cta
+    position         relative
+    border-radius    .5rem
+    background-color rgba(255, 255, 255, 0.05)
+    padding          1rem
+    width            80%
+    max-width        36rem
+    margin-top       2rem
+
+    +medium-screen()
+      display none
+
+    .pho-viewer-noviewer-cta-cross
+        position absolute
+        top      1rem
+        right    1rem
+        cursor   pointer
+
+    h3
+        margin 0 0 1.1rem 0
+
+    h3:before
+        content      ''
+        position     relative
+        top          .2rem
+        width        1.5rem
+        height       1.5rem
+        margin-right .75rem
+        display      inline-block
+        background   embedurl('./icons/icon-magic-trick.svg') center center / cover no-repeat
```

Comment:
Oh no. We still have those :'(

Reply:
Hey it IS in an `embedurl`. Can you do it with an `Icon` component instead? because in my PR about cleaning icons I removed the color from the SVG.

Reply:
Yes it can definitely be done with an Icon component. But that's for another PR.

Reply:
Can you create a card to not forget it?